Source/WebCore/crypto/keys/CryptoKeyHMAC.cpp

```diff
@@ -146,4 +147,20 @@ auto CryptoKeyHMAC::algorithm() const -> KeyAlgorithm
     return result;
 }
 
+CryptoKey::Data CryptoKeyHMAC::data() const
+{
+    auto keyData = key();
+    return CryptoKey::Data {
+        CryptoKeyClass::HMAC,
+        algorithmIdentifier(),
+        extractable(),
+        usagesBitmap(),
+        std::nullopt,
+        exportJwk(),
+        hashAlgorithmIdentifier(),
+        std::nullopt,
+        key().size() * 8,
```

Comment:
I assume this is because the key size is in bytes, and we need bits? Please make that more explicit.

Reply:
Yes; will update